jmix-translations/content/io/jmix/gridexportflowui/messages_pt_BR.properties

```diff
@@ -22,9 +22,9 @@ excelExporter.bytes=<bytes>
 excelExporter.timeFormat=h:mm
 excelExporter.dateFormat=d/m/yy
 excelExporter.dateTimeFormat=d/m/yy h:mm
-excelExporter.integerFormat=#,##0
-excelExporter.doubleFormat=#,##0.00##############
-excelExporter.decimalFormat=#,##0.00
+excelExporter.integerFormat=#.##0
```

Comment:
These are format strings for Excel, and `./,` are decimal/grouping separator placeholders. 
Your localized placeholders won't work. Perhaps you should revert this.